extensions/resource-deployment/src/helpers/optionSources.ts

```diff
@@ -0,0 +1,95 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { apiService } from '../services/apiService';
+import * as arc from 'arc';
+import { throwUnless } from '../utils';
+import * as loc from '../localizedConstants';
+import { IOptionsSource } from '../interfaces';
+import { CategoryValue } from 'azdata';
+
+
+export type OptionsSourceType = 'ArcControllersOptionsSource';
```

Comment:
the name of the type is too generic for its actual value

Reply:
Why this is an OptionsSource to get arc controllers